pom.xml

```diff
@@ -3,9 +3,9 @@
 		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>dev.cdevents.sdk-java</groupId>
+	<groupId>dev.cdevents</groupId>
 	<artifactId>cdevents-sdk-java</artifactId>
-	<version>v0.1.0-draft6</version>
+	<version>0.1.0-SNAPSHOT</version>

```

Comment:
Why do we need to change the version here?
What does "SNAPSHOT" mean in the version number?
The current version on maven central is v0.1.0-draft6 - shouldn't that match what we have here?

Reply:
It's customary to use `-SNAPSHOT` while working on a development version. Maven performs different checks when a version has this tag. This also avoids accidentally releasing a duplicate version or clobbering the same version on your Maven local repository.

Moreover, while Maven does not enforce _any_ convention at all regarding version number format it's better if we stick with semver, this means the leading `v` should not appear on the version number. However, the leading `v` may appear in the tag name applied to a release.

Git release version/tag != Maven version

Reply:
Cool, sounds good, thank you

Reply:
This is the standard I've seen in our organization when java developers are preparing for a production release. Never fully knew the depth of why it was enforced but at a high level it runs through a different series of checks. Thanks for making this contribution!